tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -185,7 +202,7 @@ def _check_if_openapi_generated_obj_is_valid(self, obj) -> Any:
 
         return obj_api
 
-    def test_selection_config(self):
+    def xtest_selection_config(self):
```

Comment:
Because I'm not really sure if this is necessary. It seems that this tests the generated code and I don't think it makes sense. Can we remove it?

Reply:
I believe it is there because we had issues with configs not being properly serialized/deseralized. If this is not an issue with the new generator we can remove the test.

Reply:
I kept the test and changed this to `DockerWorkerConfigV3`. It's not an issue.